ingest/Snakefile

```diff
@@ -36,6 +36,7 @@ else:
 rule clean:
     params:
         targets = [
+            ".snakemake",
```

Comment:
Nice, makes sense if you want to nuke everything and start the workflow from scratch!